README.md

```diff
@@ -259,43 +259,67 @@ import dialtone from "@dialpad/dialtone/eslint-plugin"
 
 #### Dialtone icons
 
-⚠️ *You should avoid importing the icons directly*, please use `DtIcon` vue component instead. ⚠️
+- Importing for Vue 2:
 
-In case you cannot use vue components, import the files directly under the `dist/` folder as following:
+```js
+// Named import
+import { DtIconArrowUp } from '@dialpad/dialtone-icons/vue2'; 
+
+// Default import (Prefered if using webpack as it is tree-shakeable by default)
+import DtIconArrowUp from '@dialpad/dialtone-icons/vue2/arrow-up';
```

Comment:
So is the first style `import { DtIconArrowUp } from '@dialpad/dialtone-icons/vue2';` tree shakable by vite? or is it not tree shakable by anything?

Reply:
Yes, `import { DtIconArrowUp } from '@dialpad/dialtone-icons/vue2';` is tree shakeable by vite